src/components/ListItem.jsx

```diff
@@ -61,6 +62,18 @@ export function ListItem({ name, items, listToken }) {
 		}
 	};
 
+	const handleAriaForColors = () => {
+		if (items?.sortedList <= 7) {
+			return 'soon';
+		} else if (items?.sortedList > 7 && items?.sortedList < 30) {
+			return 'kind of soon';
+		} else if (items?.sortedList >= 30 && items?.sortedList < 60) {
+			return 'not soon';
+		} else if (items?.sortedList > 60) {
+			return 'inactive';
+		}
+	};
```

Comment:
What does this do? I didn't see anywhere `sortedList` was assigned to `items` nor does it show up in the browser's HTML

Reply:
You will find the  sortedList variable in the other task, which takes the items (data) and makes the getDaysBetweenDates calculation and the sorting.
The comparePurchaseUrgency function is in the firebase.jsx file and imported into the List component so that Debbie can use it:
`const sortedList = comparePurchaseUrgency(data);`

Reply:
@Mama-simba I mean the code `items?.sortedList`. There's no property called `sortedList` in the context of the `items` variable in the code I highlighted.

Reply:
I added this for accessibility. I added this in the aria-label attribute on line 87. ARIA is a set of attributes you can add to HTML elements that define ways to make web content and applications accessible to users with disabilities who use assistive technologies. Hope that resolves this. :)